src/Twig/ImageExtension.php

```diff
@@ -44,6 +46,103 @@ public function getFunctions()
         ];
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    public function getFilters()
+    {
+        return [
+            new TwigFilter('inline_images', [$this, 'inlineImagesInHtmlBlock']),
+        ];
+    }
+
+    public function inlineImagesInHtmlBlock(TwigMarkup $block): TwigMarkup
+    {
+        $hasMatches = preg_match_all('/<img(?:\s+(?P<attrs>.*?))?\\/?>/i', $block, $imageMatches, PREG_SET_ORDER | PREG_OFFSET_CAPTURE);
```

Comment:
Omdat je `/` wilt matchen moet je deze excapen (i.e. `\\/`) omdat je `/` gebruikt om aan te duiden wanneer je regexp klaar is en je met de regext-flags begint.  Dat is niet heel erg leesbaar.  Maar je mag zelf kiezen welk karacter hiervoor gebruikt... dus ik zou iets anders gebruiken, bijvoorbeeld `#` of `|`

i.e. `'#<img(?:\s+(?P<attrs>.*?))?/?>#i'`

Reply:
Zelf probeer ik altijd `/` te gebruiken omdat dat de "standaard" is, tenzij ik wel heel veel moet gaan escapen (in URLs bijvoorbeeld of bij heel veel HTML). Ik vind het hier wel meevallen, dus ik laat het graag zo.